aiida/cmdline/commands/cmd_calcjob.py

```diff
@@ -151,12 +154,15 @@ def calcjob_outputcat(calcjob, path):
         )
 
     try:
-        content = retrieved.get_object_content(path)
+        # When we `cat`, it makes sense to directly send the output to stdout as it is
+        with retrieved.open(path, mode='rb') as fhandle:
+            copyfileobj(fhandle, sys.stdout.buffer)
     except (IOError, OSError) as exception:
-        # Incorrect path or file not readable
-        echo.echo_critical('Could not open output path "{}". Exception: {}'.format(path, exception))
-    else:
-        echo.echo(content)
+        # The sepcial case is error No. 32 breakon pipe, which is usually OK.
+        # It can happen if the output is redirected, for example, to `head`.
+        if exception.args[0] != 32:
```

Comment:
Probably you want to do
```python
import errno

exception.errno != errno.EPIPE
```
See reference [here](https://docs.python.org/3/library/exceptions.html#OSError.errno) for the `errno` of OSError, and you can check the errno module to verify `EPIPE` is 32

Reply:
good point